### ISSUE TITLE
Unsaved changes indication

### ISSUE THREAD

**Issue by PeterKottas** [gilmarsquinelato/i18n-manager#401]
Would be good to have unsaved changes indication (usually a "*" attached to the project name in the top bar) and also a prompt for unsaved changes if you try to quit without saving.

**Comment by gilmarsquinelato** [comment]
Yeah, this is really important. On MacOS, Electron puts this information but seems that not on other platforms. Will need to investigate how can be achieved on other platforms.
This feature is already implemented, just need to know why isn't working on other platforms 😄

**Comment by PeterKottas** [comment]
Ah, I see multiplatform is always a nightmare. Hope it won't cause you too much headache ;-)

**Comment by gilmarsquinelato** [comment]
Indeed 😄 
No worries, even getting a headache this is a requirement for that kind of app.
This is more a bug than a feature 😄
